x-pack/plugin/esql/src/main/generated-src/org/elasticsearch/xpack/esql/expression/function/scalar/math/RoundToDouble.java

```diff
@@ -102,4 +92,200 @@ static double process(double field, @Fixed double p0, @Fixed double p1, @Fixed d
         }
         return p3;
     }
+
+    @Evaluator(extraName = "5")
+    static double process(double field, @Fixed double p0, @Fixed double p1, @Fixed double p2, @Fixed double p3, @Fixed double p4) {
```

Comment:
Could you add a comment to all of the `process` methods that say that it's basically a hand unrolled binary search?